HHS/Head-Start-TTADP
# [Prod] Add "tta type" filter and submitted date

## Description of change
- Add a "tta type" filter to the activity reports landing page and the regional dashboard.
- The approved AR and CSV export now display submitted date
- You can no longer change "not started" goals to "in progress" via the RTR dropdown

## How to test
- Add the filter to a page. Verify the data is correct.
- Verify the submitted date displays on AR's
- Verify that the "in progress" option no longer appears on "not started" goals in the RTR dropdown

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1151
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1347
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1419

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
